nimare/stats.py

```diff
@@ -275,27 +275,90 @@ def nullhist_to_p(test_values, histogram_weights, histogram_bins):
     return p_values
 
 
-def fdr(p, q=0.05):
-    """Determine FDR threshold given a p value array and desired false discovery rate q.
+def fdr(p_values, q=0.05, method="bh"):
+    """Perform FDR correction on p values.
+
+    .. versionchanged:: 0.0.12
+
+        Replace thresholding function with one that calculates adjusted p values.
 
     Parameters
     ----------
-    p : 1D :class:`numpy.ndarray`
-        Array of p-values.
+    p_values : :obj:`numpy.ndarray`
+        Array of p values.
     q : :obj:`float`, optional
-        False discovery rate in fraction form. Default is 0.05 (5%).
+        Alpha value. Default is 0.05.
+    method : {"bh", "by"}, optional
+        Method to use for correction.
+        Either "bh" (Benjamini-Hochberg :footcite:p:`benjamini1995controlling`) or
+        "by" (Benjamini-Yekutieli :footcite:p:`benjamini2001control`).
```

Comment:
could you explain "by" in the other function signatures?

Reply:
Good point. I'll do that, although I'll mostly be parroting statsmodels' documentation there, since I'm not very familiar with the different approaches.

Reply:
I've added a bit more info to the associated method docstrings.